src/qml/FeatureForm.qml

```diff
@@ -241,8 +241,14 @@ Page {
           property var config: ( EditorWidgetConfig || {} )
           property var widget: EditorWidget
           property var field: Field
+          property var fieldType: FieldType
+          property var relationId: RelationId
           property var constraintValid: ConstraintValid
-
+          property var relationEditorModel:  ReferencingFeatureListModel {
+              attributeFormModel: Type === 'relation' ? form.model : undefined
```

Comment:
The model passed here is the model of the parent feature, but it should probably be one for the child feature, no?

Reply:
I pass it to connect to it - so in case the parent feature changes, the `ReferencingFeatureListModel` should be updated...

Reply:
Could only the feature be passed instead?

Reply:
Not sure. Possibly I don't understand something. But isn't this QML Object loaded only once? And later needs the model to be changed via signal/slot connections?
The attributeformmodel is passed to connect it here https://github.com/opengisch/QField/pull/502/files#diff-b81b8ea3903e69deba13fb74f7211a77R104

Reply:
If you have a `ReferencingFeatureListModel::feature` property (with a setter) it should be able to bind it to a feature in QML

Here:

```
property var currentFeature: attributeFormModel.feature // or similar
```

Inside:

```
ReferencingFeatureListModel {
    feature: currentFeature
}
```

or similar (maybe `featureId` instead of feature, but having the feature with the primary key attribute will help to avoid an extra request)...

Reply:
Okay that seems to work perfectly. What I did not understand (and still have problems in understanding) is, **_when_** the values of a already created QML object are updated. Obviously it's updated automatically here, what replaces the complete triggering of feature change from the attributeformmodel... Thanks.